internal/Get-DbaADObject.ps1

```diff
@@ -114,12 +114,12 @@ Searches in the contoso domain for a ctrlb user, suppressing all error messages
 				$searchClass = [System.DirectoryServices.AccountManagement.Principal]
 			}
 		}
-		
+
 		function Get-DbaADObjectInternal($Domain, $IdentityType, $obj, $EnableException) {
 			try {
 				# can we simply resolve the passed domain ? This has the benefit of raising almost instantly if the domain is not valid
-				$Context = New-Object System.DirectoryServices.ActiveDirectory.DirectoryContext('Domain', $Domain)
-				$DContext = [System.DirectoryServices.ActiveDirectory.Domain]::GetDomain($Context)
+				New-Object System.DirectoryServices.ActiveDirectory.DirectoryContext('Domain', $Domain) | Out-Null
+				[System.DirectoryServices.ActiveDirectory.Domain]::GetDomain($Context) | Out-Null
```

Comment:
Need to add note to possibly address this when we get to writing our coding standards 😄 

We have variations of this in commands, some will do `$null = ...` while others `... | Out-Null`. I know piping in most cases adds overhead, minimal for this case but still overhead.